core/src/test/java/info/rmapproject/core/model/impl/openrdf/ORMapEventInactivationTest.java

```diff
@@ -172,8 +172,9 @@ public void testORMapEventInactivationRMapIriRMapEventTargetTypeRMapValue() thro
 			fail("could not create agent");
 		}
 		RMapLiteral desc = new RMapLiteral("This is an inactivation event");		
-		RMapRequestAgent requestAgent = new RMapRequestAgent(associatedAgent.getIri(), new URI("ark:/29297/testkey"));
-		ORMapEventInactivation event = new ORMapEventInactivation(uri2OpenRdfIri(create("http://example.org/event/1")), requestAgent,RMapEventTargetType.DISCO, desc);
+		RequestEventDetails reqEventDetails = new RequestEventDetails(associatedAgent.getIri(), new URI("ark:/29297/testkey"));
+		reqEventDetails.setDescription(desc);
```

Comment:
Could there be an additional constructor for `RequestEventDetails` that accepts a description?